### PR TITLE
Remove AUTHORS/CONTRIBUTORS file

### PR DESCRIPTION
**Pull request by rinx**
Signed-off-by: Rintaro Okamura <rintaro.okamura@gmail.com>

### Description:

- [X] Remove CONTRIBUTORS file because now we use all-contirbutors tool.
- [x] Remove AUTHORS file
- [X] Revise format of all-contributors tool.
    - https://allcontributors.org/docs/en/bot/configuration
    - https://contributors-img.web.app/ may be useful?
    - restrict number of icons for each contributor
        - main 4 icons are applied to each contributor
- removed useless badges

### Related Issue:

#414

### How Has This Been Tested?:

nothing

### Environment:

- Golang Version: 1.14.3
- Docker Version: 19.03.8
- Kubernetes Version: 1.18.2
- NGT Version: 1.11.5

### Types of changes:

- [ ] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [X] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [X] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [X] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [X] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -12,23 +12,9 @@
       "profile": "http://kpango.com",
       "contributions": [
         "code",
-        "design",
```
Comment (kpango) [inline .all-contributorsrc:15]: design is better than ideas

File: README.md [hunk 2]
```diff
@@ -2,18 +2,15 @@
 <img src="./assets/image/svg/logo.svg" width="50%">
 </div>
 
-[![License: Apache 2.0](https://img.shields.io/badge/License-Apache2-blue.svg?style=flat-square)](https://opensource.org/licenses/Apache-2.0)
-[![CLA](https://cla-assistant.io/readme/badge/vdaas/vald?&style=flat-square)](https://cla-assistant.io/vdaas/vald)
+[![License: Apache 2.0](https://img.shields.io/github/license/vdaas/vald.svg?style=flat-square)](https://opensource.org/licenses/Apache-2.0)
 [![release](https://img.shields.io/github/release/vdaas/vald.svg?style=flat-square)](https://github.com/vdaas/vald/releases/latest)
+[![GoDoc](https://img.shields.io/badge/godoc-reference-blue.svg?style=flat-square)](https://godoc.org/github.com/vdaas/vald)
```
Comment (kpango) [inline README.md:7]: recently go doc's sites were moved to pkg.go.dev
```suggestion
[![GoDoc](https://img.shields.io/badge/godoc-reference-blue.svg?style=flat-square)](https://pkg.go.dev/github.com/vdaas/vald)
```

File: .all-contributorsrc (hunk 1 above)
Comment (rinx) [reply .all-contributorsrc:15]: updated

**Pull request merged**